test/e2e/rmi_test.go

```diff
@@ -282,6 +282,5 @@ RUN find $LOCAL
 		session := podmanTest.PodmanNoCache([]string{"image", "rm"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(125))
-		Expect(session.LineInOutputContains("image name or ID must be specified"))
```

Comment:
Just wondering why you removed this Expect rather than adding the `.To(BeTrue())` bit to it?

Reply:
The stdout from the session is empty (I'm guessing because the ExitCode is nonzero), so the statement ends up failing. It does look like there is an `ErrorGrepString()` function I could try and replace it with!

Reply:
OK, I'm puzzled.  @gabibeyer you're right this is the correct change to remove it.  I'm pretty sure we required a name/id at one point but removed that.  I'm surprised this line was not failing?  So I'd leave this as you have it, thanks for the 411.

Reply:
Whoops read your response a little late. The error code is reporting correctly in `stderr`, so it looks like the `ErrorGrepString()` function does work! I can revert that change though if you would prefer the removal of the line :)
I _think_ it was passing because it was missing the 'matcher' (the `.To(BeTrue())`), so it was automatically passing, when I added the `.To(BeTrue())` it failed!

Reply:
@gabibeyer I like how you've reworked that line, thanks for going through the extra hoops there.